applications/GeoMechanicsApplication/tests/cpp_tests/test_generalized_newmark_T_scheme.cpp

```diff
@@ -140,7 +101,7 @@ KRATOS_TEST_CASE_IN_SUITE(ThermalSchemeUpdate_SetsDtTemperature, KratosGeoMechan
     node.FastGetSolutionStepValue(DT_TEMPERATURE, 1) = previous_dt_temperature;
 
     // This is the expected value as calculated by the UpdateVariablesDerivatives
-    constexpr double expected_value = 7.0/3.0;
+    constexpr double expected_value = 7.0 / 3.0;
```

Comment:
This far from the place where this variable is used. Further, writing 7.0 / 3.0 in the KRATOS_EXPECT_DOUBLE_EQ statement is equally clear.